제이_서제경/4주차/Mission2/Umc-Mission2/src/pages/MyPage.tsx

```diff
@@ -0,0 +1,3 @@
+export const Mypage = () => {
+  return <div>Mypage</div>;
+};
```

Comment:
보통은 

```suggestion
const Mypage = () => {
  return <div>Mypage</div>
}
export default Mypage
```

이런 식으로 사용합니다!